src/scripts/check-health.sh

```diff
@@ -2,6 +2,8 @@
 user="$(id -u)"
 if [ "$user" -ne 0 ]; then export SUDO="sudo"; else export SUDO=""; fi
 if grep "Alpine" < /etc/issue > /dev/null 2>&1; then
+    #shellcheck disable=1090
+    . "${BASH_ENV}"
     if [ "$(jq --version > /dev/null; echo $?)" -ne 0 ]; then
```

Comment:
Also assuming `command` is available in `ash`. Which I think it is.

```suggestion
    if ! command -v jq &> /dev/null; then
```

Reply:
jq --version works in alpine (dash) but I think command is safer. Again &> is not available so I had to use 2>&1.